packages/devtools_app/lib/src/shared/diagnostics/inspector_service.dart

```diff
@@ -189,11 +191,12 @@ abstract class InspectorServiceBase extends DisposableController
 /// Manages communication between inspector code running in the Flutter app and
 /// the inspector.
 class InspectorService extends InspectorServiceBase {
-  InspectorService()
+  InspectorService({ValueListenable<IsolateRef?>? evalIsolate})
```

Comment:
remove this parameter and pass `evalIsolate: serviceConnection.serviceManager.isolateManager.mainIsolate` directly below